### DISCUSSION TITLE
overrides main.html can not show image on a sub page

### DISCUSSION THREAD

**Discussion by obaito** [mkdocs/mkdocs#2693] [Q&A]
![image](https://user-images.githubusercontent.com/95470064/144572066-830b2b80-905b-4db2-8726-e7435399f045.png)

I want to show the google image on the top like the above.

But the problem is that when I click the subpage, the google image is broken like below.
![image](https://user-images.githubusercontent.com/95470064/144572476-52f487b3-acaf-43b2-bbb6-2b9a3fb6859f.png)

I put the google.png like this.
⇒overrides/assets/images/google.png file

And the main.html is like this.

```jinja
{% extends "base.html" %}

{% block announce %}
<div>
  I want to show a image 
  <a href="https://www.google.com/"><img src="./assets/images/google.png"></a>
  <span>on the top</span>
</div>
{% endblock %}
```

Additional, I use squidfunk/mkdocs-material:8.0.2.
Is there someone know how to sovle this problem? Thank you.

**Comment by oprypin** [comment]
I think you need to change to

```jinja
<img src="{{ "assets/images/google.png" | url }}">
```

https://www.mkdocs.org/dev-guide/themes/#base_url

**Reply by obaito** [reply]
Thank you very much !
My problem got solved !
I have to check the URL too.
